dynamic_movement_primitive/DynamicMovementPrimitiveMsg dmp
dynamic_movement_primitive/NC2010DynamicMovementPrimitiveStateMsg state
dynamic_movement_primitive/NC2010DynamicMovementPrimitiveParametersMsg parameters
dynamic_movement_primitive/NC2010TransformationSystemMsg[] transformation_systems
dynamic_movement_primitive/NC2010CanonicalSystemMsg canonical_system